app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HelloRailsReact</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
```

Comment:
I'm unable to test the app because of an error being thrown here. 
<img width="1069" alt="image" src="https://user-images.githubusercontent.com/67957533/232017106-21cffa4c-e43f-4e1d-a08f-b7f65a5030f1.png">

if it's not a problem please let the next code reviewer know how best to run the app 


- [ ] Kindly fix the issue